src/main/java/com/nylas/Participant.java

```diff
@@ -55,12 +65,14 @@ public Participant comment(String comment) {
 		Map<String, Object> params = new HashMap<>();
 		Maps.putIfNotNull(params, "name", name);
 		Maps.putIfNotNull(params, "email", email);
+		Maps.putIfNotNull(params, "phone_number", phone_number);
 		Maps.putIfNotNull(params, "comment", comment);
 		return params;
 	}
 
 	@Override
 	public String toString() {
-		return "Participant [name=" + name + ", email=" + email + ", status=" + status + ", comment=" + comment + "]";
+		return "Participant [name=" + name + ", email=" + email + ", phone_number=" + phone_number
+				+ ", status=" + status + ", comment=" + comment + "]";
 	}
```

Comment:
might want to use StringBuilder and append() instead of concat; its faster and more readable

Reply:
any tests to run for this? Just a small one to exercise the code building these objs and getting the string representations

Reply:
@jseller thanks for your feedback! I think changing this short of a concat to a StringBuilder won't offer much noticeable increase and I tried it out in terms of readability, it just gets too long for what it does. However, I have re-formatted the toString concats to be more readable, let me know if that works better! Also unfortunately we do not have any unit tests atm, but you can locally edit EventsExample.java and log a participant's toString value if that works!